CONTRIBUTING.md

```diff
@@ -36,7 +45,7 @@ The frontend code can take advantage of `MorphSVGPlugin` for more fluid SVG anim
 
 ## Logging in
 
-After the bootstrap data is loaded, you can login using the test account. Username is "admin" and password is "testing123"
+After the bootstrap data is loaded, you can login using the test account. Username is "owner" and password is "testing123"
```

Comment:
Nice catch. We could go a step further and say that the password for any of the seed users (including owner) is "testing123".

Reply:
> Nice catch. We could go a step further and say that the password for any of the seed users (including owner) is "testing123".

👍  added those.